### PR TITLE
Eng 2450 fix node layout algo

### PR DESCRIPTION
**Pull request by agiron123**
## Describe your changes and why you are making these changes
This PR includes some changes to fix our node layout algorithm's settings.

It also includes a change to fit the graph to view on load.

## Related issue number (if any)
ENG-2450

## Loom demo (if any)
https://www.loom.com/share/6eed4fcdaab64435840ded127e01915b

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [x] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [x] If this is a new feature, I have added unit tests and integration tests.
- [x] I have run the integration tests locally and they are passing.
- [x] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [x] All features on the UI continue to work correctly.
- [x] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/ui/common/src/components/workflows/ReactFlowCanvas.tsx [hunk 1]
```diff
@@ -27,11 +27,30 @@ const ReactFlowCanvas: React.FC<ReactFlowCanvasProps> = ({
...
+    return {
+      id: edge.id,
+      source: edge.source,
+      target: edge.target,
+      type: edge.type,
+      container: 'root',
+    };
+  });
+
+  const canvasNodes = nodes.map((node) => {
```
Comment (agiron123) [inline src/ui/common/src/components/workflows/ReactFlowCanvas.tsx:40]: Having some extra keys in our input nodes and edges caused some issues in how the nodes were positioned. This function simply maps our nodes to ensure that the node and edge objects have the proper shape.

File: src/ui/common/src/reducers/workflow.ts [hunk 2]
```diff
@@ -497,15 +496,9 @@ export const handleGetSelectDagPosition = createAsyncThunk<
...
           'elk.direction': 'RIGHT',
-          'elk.alignment': 'CENTER',
-          'elk.spacing.nodeNode': '80',
-          'elk.layered.spacing.nodeNodeBetweenLayers': '80',
-          // https://www.eclipse.org/elk/reference/options/org-eclipse-elk-layered-nodePlacement-strategy.html
-          'nodePlacement.strategy': 'NETWORK_SIMPLEX',
-          'org.eclipse.elk.edgeRouting': 'SPLINES',
-          //https://www.eclipse.org/elk/reference/options/org-eclipse-elk-layered-nodePlacement-strategy.html
-          'crossingMinimization.strategy': 'INTERACTIVE',
-          'crossingMinimization.forceNodeModelOrder': true,
```
Comment (agiron123) [inline src/ui/common/src/reducers/workflow.ts:508]: Removing this allows ordering of nodes to not matter when rendering the graph. This was what caused the 'x' to show in one of the churn workflows, but not the other.

File: src/ui/common/src/reducers/workflow.ts [hunk 3]
```diff
@@ -497,15 +496,9 @@ export const handleGetSelectDagPosition = createAsyncThunk<
...
+          'elk.spacing.nodeNode': '300',
+          'elk.layered.spacing.nodeNodeBetweenLayers': '300',
+          'elk.layered.nodePlacement.strategy': 'INTERACTIVE',
```
Comment (agiron123) [inline src/ui/common/src/reducers/workflow.ts:501]: Interactive yields the best looking graphs with elk. The other graphs still had quite a bit of edge crossing.

File: src/ui/common/src/components/workflows/nodes/CheckOperatorNode.tsx [hunk 4]
```diff
@@ -92,9 +92,9 @@ const CheckOperatorNode: React.FC<Props> = ({ data, isConnectable }) => {
         color: textColor,
         borderColor: borderColor,
         '&:hover': { backgroundColor: hoverColor },
-        minHeight: 'unset',
-        minWidth: '240px',
-        padding: '0px',
+        //   //minHeight: 'unset',
```
Comment (agiron123) [inline src/ui/common/src/components/workflows/nodes/CheckOperatorNode.tsx:95]: remove comments.

**Review by likawind — COMMENTED** [review]
Codes LGTM, @vsreekanti I will let you make the final call if the looking is good to go

**Review by likawind — APPROVED** [review]

**Pull request merged**